SPECS/golang/golang.spec

```diff
@@ -12,7 +12,7 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           golang
-Version:        1.17.8
+Version:        1.18.3
```

Comment:
Do we also need to bump the packages that are built with GoLang (since these are all statically linked?)

Reply:
For the last change to `golang` we did not do it. I assumed that was the default.
Should I bump the dependencies ?

Reply:
we might have missed it for the latest bump but we usually do it. I think Pawel has a tool for that